css/grid/grid-pro.css

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license Highcharts Grid Pro v@product.version@
+ *
+ * (c) 2009-2025 Highsoft AS
+ *
+ * License: www.highcharts.com/license
+ */
+
+/* Import Grid Lite styles */
+@import url("grid-lite.css");
```

Comment:
IMO, there are a few problems with it
- It loads grid-lite.css from the same directory, but when the user uses only PRO version, the reference is undefined
- We could use the full path, but in terms of local loading files, it will be broken (user should replace this import with their path) 

Maybe, we should consider using a gulp task to copy all declarations from grid-lite and paste them here.